migtests/tests/pg/datatypes/validate

```diff
@@ -46,9 +46,22 @@ EXPECTED_SUM_OF_COLUMN = {
 EXPECTED_ENUM_VALUES = ['mon', 'tue', 'wed', 'thu', 'fri', 'sat', 'sun']
 EXPECTED_BYTE_TYPE_VALUES = ['3031303130', '2d61626364', '34343538']
 
+#Due to the large sized source database requirements all tables will come up as colocated for now
```

Comment:
yeah, we need to find a way so that those size limits in ExperimentDB can be modified for the test case.
Maybe add a hidden flag and provide a updated ExpDB

Reply:
Yes that would be the best case scenario.